.github/workflows/ci.yml

```diff
@@ -0,0 +1,40 @@
+name: CI Build
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      ARTIFACTORY_RELEASE_REPO: eaiw-release-local
+      ARTIFACTORY_RESOLVE_REPO: eaiw-local
+      ARTIFACTORY_SNAPSHOT_REPO: eaiw-snapshot-local
+      ARTIFACTORY_CONTEXT: ${{ secrets.ARTIFACTORY_CONTEXT }}
+      ARTIFACTORY_PUBLISH_USER: ${{ secrets.ARTIFACTORY_PUBLISH_USER }}
+      ARTIFACTORY_PUBLISH_PASSWORD: ${{ secrets.ARTIFACTORY_PUBLISH_PASSWORD }}
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Cache Gradle packages
+        uses: actions/cache@v2
+        with:
+          path: ~/.gradle/caches
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle') }}
+          restore-keys: ${{ runner.os }}-gradle
```

Comment:
This is the standard caching action recommended by github: https://docs.github.com/en/free-pro-team@latest/actions/guides/building-and-testing-java-with-gradle#caching-dependencies